src/win32.mak

```diff
@@ -377,8 +380,11 @@ pvs:
 #	$(PVS) --cfg PVS-Studio.cfg --cl-params /I$C;$(TK) /Tp $(BACKSRC) --source-file $(BACKSRC)
 #	$(PVS) --cfg PVS-Studio.cfg --cl-params /I$(TK) /Tp $(TKSRCC) --source-file $(TKSRCC)
 
-checkwhitespace:
-	$(HOST_DC) -run checkwhitespace $(SRCS) $(GLUESRC) $(ROOTSRC)
+checkwhitespace: $(TOOLS_DIR)
+	$(HOST_DC) -run $(TOOLS_DIR)\checkwhitespace $(SRCS) $(GLUESRC) $(ROOTSRC)
+
+$(TOOLS_DIR):
+	git clone --depth=1 $(GIT_HOME)/tools $@
```

Comment:
Biggest problem with Windows makefiles is DigitalMars make, which is very simplistic. Making it use GNU make will introduce a dependency and will likely cause headaches with weird errors depending on which version of GNU Make for Windows / Unix-on-Windows toolset you use. (There is a lot more to it than make.exe.)